src/steps/command/plugins.ts

```diff
@@ -33,4 +35,10 @@ export class PluginsImpl<T> extends Chainable<T> implements Plugins<T> {
         this.plugins.push(plugin);
         return this.parent;
     }
+
+    filter(
+        ...args: Parameters<Plugins<T>['filter']>
+    ): ReturnType<Plugins<T>['filter']> {
```

Comment:
As above, I think the signature here should be explicit.

Using this index approach would be useful if the type was very complex or likely to change, but the type is quite simple and will likely never change